frontend/src/actions/config.ts

```diff
@@ -1,10 +1,14 @@
 export const config = {
+  BASE_URL:
+    process.env.REACT_APP_ENV === 'local'
+      ? 'http://127.0.0.1:8000'
+      : 'https://ervin-els-backend.herokuapp.com',
   URL:
     process.env.REACT_APP_ENV === 'local'
       ? 'http://127.0.0.1:8000/api'
       : 'https://ervin-els-backend.herokuapp.com/api',
   IMG_URL:
     process.env.REACT_APP_ENV === 'local'
       ? 'http://127.0.0.1:8000'
-      : 'https://ervin-els-backend.herokuapp.com/api',
+      : 'https://ervin-els-backend.herokuapp.com',
```

Comment:
Your config is growing bigger and having duplicate lines, it is time to refactor this.